website/content/docs/upgrading/identity-deduplication.mdx

```diff
@@ -0,0 +1,406 @@
+    capabilities = ["read", "create", "update"]
+}
+```
+
+Then before the rename they would be able to read and write key-value pairs
+(KVs) with the prefix `kv/users/Janine/`. After the rename, the user would
+ instead be granted access to KVs with prefix `kv/users/Janine-<UUID>/`, however
+the KV mount doesn't know anything special about either path and so any existing
+KVs the user had stored would remain at the old paths and `Janine-<UUID>` would
+no longer be able to access them.
+
+You can resolve this in two ways:
+ 1. **Change or add policy**. You could define a new policy or a new rule in a
+user-specific policy that explicitly granted access back to the old resource
+path. For example you could add the following to a policy specific to the
+renamed `Janine-UUID` entity:
+  ```hcl
+  path "kv/users/Janine/*" {
+    capabilities = ["read", "create", "update"]
+  }
+  ```
+ 2. **Move the resources.** In some cases operators could move the resources to
+be at the new path via the API. For example KV secrets could be renamed to match
+the new path. This might be arduous if there are many to rename, and for some
+other resource types it might not be possible to move them for other reasons
+e.g. external resources expecting access at that path. You should decide whether
+you perform any migration before or after activating the flag. If you choose to
+do it afterwards, the resources will be unavailable until the migration is
+complete. If you choose to do it first then the resources might be unavailable
+until the flag is activated if moved, or you might need to take care to keep
+updates to both old an new paths in sync until the flag is switched.
+
+#### Resolving external references (e.g. Terraform)
+
+It it is possible that renaming resources could break external references if
+they refer directly to the entity or group by name.
+
+For example, if you have a Terraform configuration like this:
+
+```hcl
+terraform {
+  required_providers {
+    vault = {
+      source = "hashicorp/vault"
+    }
+  }
+}
+
+provider "vault" {}
+
+resource "vault_identity_entity" "BOB" {
+  name              = "BOB"
+  policies          = ["TEST"]
+}
+
+
+resource "vault_identity_entity" "bob" {
+    name              = "bob"
+    policies          = ["test"]
+}
+```
+
+By default this Terraform config would not work correctly since Vault would
+match the names in a case-insensitive way and reject creating the second
+resource. But if you have had duplicates reported in your Vault cluster, that
+means that due to some historical issue Vault is running in a mode that would
+allow both of these case variations to exist a separate entities.
+
+After activating the `force-identity-deduplication` flag, one of these entities
+will no longer have the specified name and so Terraform will attempt to rename
+it and be unable to because that violates the case-insensitive name constraint
+that has be re-introduced. An apply would result in an error such as:
```

Comment:
"that has been reintroduced in more recent versions of Vault", right? (for clarity and a typo there)